Multisig.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -262,15 +262,6 @@
           "version": "2.0.1"
         }
       },
-      {
```

Comment:
Do you want to return this back after updating the library?

Reply:
hm,  yes, somehow this was deleted